windows/winpublic.cpp

```diff
@@ -35,6 +35,16 @@ void uiWindowsEnsureMoveWindowDuringResize(HWND hwnd, int x, int y, int width, i
 		logLastError(L"error moving window");
 }
 
+void uiWindowsResizeWindow(HWND hwnd, int width, int height) {
```

Comment:
I'm pretty sure these two are provided in winutil.cpp anyway.

Will review the rest of the code later.

Reply:
There was `uiWindowsEnsureMoveWindowDuringResize`, but that moves and resizes the window. I added these functions because when you either move or resize a window, Windows doesn't give accurate values for the other two, so if you want it to move, it'll end up resizing also. Which is why I made these with the `SWP_NOSIZE` and `SWP_NOMOVE` flags.